/**
 * Simplifies `error` handling in `try...catch` statements.
 *
 * JavaScript is weird, you can `throw` anything. Since it's possible for
 * library authors to throw something unexpected, we have to take precautions.
 */
export declare function getErrorMessage(error: unknown, fallbackMessage?: string): string;
//# sourceMappingURL=error.d.ts.map